packages/assets-controllers/src/TokenDetectionController.ts

```diff
@@ -83,6 +90,7 @@ export class TokenDetectionController extends BaseController<
       getTokensState,
       getNetworkState,
       getPreferencesState,
+      getNetworkClientById,
```

Comment:
This is a breaking change. Can you update the PR description so it's more visible?